sonar-plugin-api/src/main/java/org/sonar/api/batch/sensor/issue/NewIssue.java

```diff
@@ -100,6 +101,21 @@ enum FlowType {
    */
   NewIssueLocation newLocation();
 
+  /**
+   * Create a new quick fix
+   * @return a new uninitialized instance of a quick fix for a given issue
+   * @since 9.13
+   */
+  NewQuickFix newQuickFix();
```

Comment:
Based on the javadoc, it's unclear to me if the quick fix needs to be added using `addQuickFix`.

Reply:
I made the description clearer, thanks! I am not a big fan of having a `new` and a `add`, we discussed it when we introduced that API in `sonarlint-plugin-api` and preferred to be consistent with other APIs from this class (like `newLocation` and `addLocation` methods).